web/frontend/src/components/movement-sensor/index.svelte

```diff
@@ -274,6 +313,33 @@ useConnect(() => {
               {compassHeading?.toFixed(2)}
             </td>
           </tr>
+          {#if accuracy?.compassDegreesError}
```

Comment:
Same here - why is this being added to the compass table?

Reply:
This is a compass heading related accuracy. If the driver finds the compass heading drifting, we want to know by how much. Would you recommend a different place for it?

Reply:
Compass Degrees Error value should indicate how erroneous the compass heading is. This is useful information for user to debug their hardware or know when to calibrate it.